docs/reference/two_step_validation.rst

```diff
@@ -35,5 +35,38 @@ Edit the configuration file:
         google_authenticator:
             enabled: true
             server:  yourserver.com
+            ip_white_list:
+                - 127.0.0.1
+            forced_for_role:
+                - ROLE_ADMIN
+
+Also, if you want to use ``ip_white_list`` and ``forced_for_role`` configuration for automatically setting secret
+to user and showing QR-code in login form, you need to set success handler in your firewall to ``sonata.user.google.authenticator.success_handler``, example:
```

Comment:
What does "setting secret to user" mean here exactly? I'm a noob.

Reply:
"to set the success handler"

Reply:
@greg0ire the secret
to user (secret - a connection between user and device that will scans QR-code)